-------------------------------------------------------------------------------
Test set: br.upe.ResultadoMegasenaTest
-------------------------------------------------------------------------------
Tests run: 1, Failures: 0, Errors: 1, Skipped: 0, Time elapsed: 1.229 s <<< FAILURE! - in br.upe.ResultadoMegasenaTest
testObtemUltimoResultado(br.upe.ResultadoMegasenaTest)  Time elapsed: 1.155 s  <<< ERROR!
java.lang.RuntimeException: Um erro inesperado ocorreu.
	at br.upe.ResultadoMegasenaTest.testObtemUltimoResultado(ResultadoMegasenaTest.java:17)
Caused by: org.apache.http.client.HttpResponseException: status code: 400, reason phrase: Bad Request
	at br.upe.ResultadoMegasenaTest.testObtemUltimoResultado(ResultadoMegasenaTest.java:17)

